Lib/mutatorMath/objects/mutator.py

```diff
@@ -133,7 +133,7 @@ def getAxisNames(self):
         s = {}
         for l, x in self.items():
             s.update(dict.fromkeys([k for k, v in l], None))
-        return s.keys()
+        return sorted(list(s.keys()))
```

Comment:
sorry but.. why does it have to be a *sorted* list? Why incur in the (however tiny) overhead? Even the python2's dict.keys(), which returns a list, does not guarantee it to be sorted (unless one uses an `OrderedDict`).

I think we want to only sort when it is needed (in the doctest), and not in the method itself.
Who cares the list of axis names is sorted alphabetically..